jobsdb/jobsdb.go

```diff
@@ -319,6 +319,7 @@ jobs. The caller must call the SetUp function on a HandleT object
 */
 type HandleT struct {
```

Comment:
I would argue that some kind of short documentation on the fields of important structs as such, could be useful. Especially for a newcomer like myself. That being said, I also understand that providing concise documentation might not always be the easiest thing!